DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -64,15 +64,36 @@ FROM Centres
             return centres;
         }
 
-        public Centre GetCentreDetailsById(int centreId)
+        public Centre? GetCentreDetailsById(int centreId)
         {
-            return connection.QueryFirstOrDefault<Centre>(
-                @"SELECT c.CentreID, c.CentreName, c.RegionID, r.RegionName
+            var centre = connection.QueryFirstOrDefault<Centre>(
+                @"SELECT c.CentreID,
+                            c.CentreName,
+                            c.RegionID,
+                            r.RegionName,
+                            c.NotifyEmail,
+                            c.BannerText,
+                            c.SignatureImage,
+                            c.CentreLogo
                         FROM Centres AS c
                         INNER JOIN Regions AS r ON r.RegionID = c.RegionID
                         WHERE CentreID = @centreId",
                 new { centreId }
             );
+
+            if (centre?.CentreLogo?.Length < 10)
+            {
+                centre.CentreLogo = null;
+            }
+
+            if (centre?.CentreLogo == null)

```

Comment:
Oh this was meant to be logged only if centre == null (if we can't find it per id) 🙏

Reply:
Good point, think I was having one of those days yesterday. Fixed now.